### PR TITLE
Go bindings: add missing error codes

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: bindings/go/evmc/evmc.go [hunk 1]
```diff
@@ -120,7 +126,7 @@ func (err Error) Error() string {
 		return fmt.Sprintf("evmc: unknown internal error (%d)", int32(code))
```
Comment (chfast) [inline bindings/go/evmc/evmc.go]: Maybe remove "unknown" from here as we do not predefine any internal errors at the moment.

Comment (axic) [reply bindings/go/evmc/evmc.go]: Perhaps "specific internal error" ? "user-define internal error" ?

Comment (chfast) [reply bindings/go/evmc/evmc.go]: I'd go with "internal error" only.

File: bindings/go/evmc/evmc.go [hunk 2]
```diff
@@ -110,6 +110,12 @@ func (err Error) Error() string {
 		return "evmc: precompile failure"
 	case C.EVMC_CONTRACT_VALIDATION_FAILURE:
 		return "evmc: contract validation failure"
+	case C.EVMC_ARGUMENT_OUT_RANGE:
```
Comment (chfast) [inline bindings/go/evmc/evmc.go]: Should be `EVMC_ARGUMENT_OUT_OF_RANGE`. See Go CI builds.

Comment (axic) [reply bindings/go/evmc/evmc.go]: Fixed already, just didn't push before the other one is decided.

File: bindings/go/evmc/evmc.go (hunk 1 above)
Comment (axic) [reply bindings/go/evmc/evmc.go]: Then the other `internal error` should also have a value IIRC.

**Pull request merged**